blocks/microsoft/version.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2023042400;
+$plugin->version = 2023042420;
```

Comment:
I'm a bit afraid of changing the version of the plugin, as it might also be changed independently upstream. Is it a problem if we just keep it as-is?